backend/src/main/java/de/neuefische/ffmjava221/teamprojekt/backend/meals/MealRepository.java

```diff
@@ -22,8 +22,11 @@ public Meal addMeal(Meal newMeal) {
     }
 
     public Meal updateMeal(int index, Meal meal) {
-       meals.set(index, meal);
-       return meal;
+        meals.set(index, meal);
+        return meal;
     }
 
-}
+    public Meal deleteMeal(int index) {
+        return meals.remove(index);
+    }
```

Comment:
Maybe you can throw an exception here with an message | maybe an NoSuchElementEx..

Reply:
We prevented that .remove(-1) can happen

Reply:
ok ^^